README.md

```diff
@@ -17,6 +17,30 @@ Azure Toolkits for Java is an open-source project that helps Java developers eas
 * [Java Developer Center on Azure](https://docs.microsoft.com/en-us/java/azure/)
 * [Get Started for HDInsight](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-apache-spark-intellij-tool-plugin)
 
+### Building
+* Clone the repository with HTTPS or SSH:
+    ```
+    $ git clone  https://github.com/Microsoft/azure-tools-for-java.git
+    ```
+* Run the following command under the Project base path:
+    ```
+    $ mvn clean install -f Utils/pom.xml
+    ```
+* Then you can use gradle to build the plugin
+    ```
+    $ ./gradlew -b PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle buildPlugin
+    ```
+    You can find the outputs under ```PluginsAndFeatures\azure-toolkit-for-intellij\build\distributions```
+* Or run the IntelliJ instance:
```

Comment:
confuse here. It is run not build right?

Reply:
Yes, I'll remove this, since run/debug session will cover this task